lib/alaveteli_localization.rb

```diff
@@ -1,8 +1,35 @@
-require 'alaveteli_localization/locale'
-require 'alaveteli_localization/hyphenated_locale'
-require 'alaveteli_localization/underscorred_locale'
-
 class AlaveteliLocalization
+  class Railtie < Rails::Railtie
+    config.before_configuration do
+      require 'alaveteli_localization/locale'
+      require 'alaveteli_localization/hyphenated_locale'
+      require 'alaveteli_localization/underscorred_locale'
+
+      require 'routing_filters'
+      require 'i18n_fixes'
+
+      paths = ['locale']
+      paths << 'locale_alaveteli_pro' if AlaveteliConfiguration.
+        enable_alaveteli_pro
+
+      repos = paths.map do |path|
+        FastGettext::TranslationRepository.build('app', path: path, type: :po)
+      end
+      AlaveteliLocalization.set_default_text_domain('app', repos)
+
+      I18n::Backend::Simple.send(:include, I18n::Backend::Fallbacks)
+
+      AlaveteliLocalization.set_default_locale_urls(
+        AlaveteliConfiguration.include_default_locale_in_urls
+      )
```

Comment:
There was some minor refactoring here but should be equivalent to what was in `config/initializers/fast_gettext.rb`

Reply:
Is!